src/EventTimer.ts

```diff
@@ -66,8 +74,32 @@ export class EventTimer {
 		return this.init();
 	}
 
+	/**
+	 * Returns all commercial timers. CMP-related timers are not tracked
+	 * by EventTimer so they need to be concatenated to EventTimer's private events array.
+	 */
+	public get events(): Event[] {
+		return typeof window.performance !== 'undefined' &&
+			'mark' in window.performance
```

Comment:
Just checking again and noticed this should be `'getEntriesByName' in window.performance`

Reply:
I think you can use optional chaining here:

```ts
window.performance?.mark ? 'a' : 'b';
```

Reply:
We tried doing this but the compiler thinks `window.performance` is always defined.